src/app/merge-duplicates/merge-duplicates-table/merge-duplicates-table.component.html

```diff
@@ -8,13 +8,17 @@
         <span *ngIf="type === 'contacts'">{{ 'captions.Contact.uuid' | translate }}</span>
       </th>
       <td mat-cell *matCellDef="let element ; let index = index" class="uuid" [ngClass]="{'pl-child': isChild(index)}">
-        <mat-icon
-          *ngIf="!isChild(index)"
-          [ngClass]="{'rotate': isChildHidden(index + 1)}"
-          (click)="hideChild(index)"
-          class="material-icons-round show-more"
-        >arrow_downward</mat-icon>
-        <a [routerLink]="['/cases/case/' + element.uuid]" mat-icon-button>
+        <div class="show-more-content" *ngIf="!isChild(index)">
+          <mat-icon
+            [ngClass]="{'rotate': isChildHidden(index + 1)}"
+            (click)="hideChild(index)"
+            class="material-icons-round show-more"
+          >arrow_downward</mat-icon>
+        </div>
+        <a *ngIf="type === 'cases'" [routerLink]="['/cases/case/' + element.uuid + '/details']" mat-icon-button>
```

Comment:
We could use some constants for this in common.ts like it is done for SPECIFIC_SEARCH_TYPE.